__tests__/app.test.js

```diff
@@ -1,15 +1,44 @@
 const endpointsJson = require("../endpoints.json");
-/* Set up your test imports here */
-
-/* Set up your beforeEach & afterAll functions here */
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const app = require("../app");
+beforeEach(() => seed(data));
+afterAll(() => db.end());
 
 describe("GET /api", () => {
-  test.skip("200: Responds with an object detailing the documentation for each endpoint", () => {
+  test("200: Responds with an object detailing the documentation for each endpoint", () => {
     return request(app)
       .get("/api")
       .expect(200)
-      .then(({ body: { endpoints } }) => {
-        expect(endpoints).toEqual(endpointsJson);
+      .then(({ body }) => {
+        expect(body).toEqual(endpointsJson);
+      });
+  });
+});
+
+describe.only("GET /api/topics", () => {
+  test("should return all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
+            img_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("should return 404 for non-existent routes", () => {
+    return request(app)
+      .get("/api/notAValidUrl")
+      .expect(404)
```

Comment:
This test is great, but as it's not specific to the `api/topics` endpoint, it should be in it's own describe block for non-existent routes 👍